openfasoc/generators/common/__init__.py

```diff
@@ -7,6 +7,9 @@
 	2. `COMMON_PLATFORMS_PREFIX_MAP` (dict): This is a dictionary of common platforms (currently sky130) and their cell naming prefixes.
 - `common.simulation`
 	1. `run_simulations()`: Used to run SPICE testbenches with multiple parameters.
-
+- `common.check_gen_files`
+	1. `check_gen_files(parameters: json_filename, is_tempsense)`: Used to check if the various files that should be generated by the flow are present in their required directories. 
+- `common.check_gen_extensions`
```

Comment:
I don't see any python file with this name in this directory. Can you please more about this?

Reply:
the check_gen_files.py file exists in the `generators/common` directory, and it has the check_gen_files function
Each generator (temp-sense, ldo-gen, cryo) has check_gen_extensions files in their `tools` directories which contains the extensions of the files generated by the flow

Reply:
right but why are you adding that function as `common.check_gen_extensions`. That doesn't belong to the common library, does it?